3scale/porta
# Adding ApiDocs proxy to OAS v3

**What this PR does / why we need it**:

**ActiveDocs** is now using **swagger-ui v3** with **OAS 3**. But we are missing the ApiDocs proxy implementation (proxy is implemented only for v2), it results in a CORS issue when trying to make http requests:

![cors](https://user-images.githubusercontent.com/13486237/96862214-75784200-1465-11eb-8ce4-76362fc1250b.png)


**Which issue(s) this PR fixes** 

https://issues.redhat.com/browse/THREESCALE-5736

**Verification steps** 

- Go to ActiveDocs
- Create a valid OAS3 spec or use the default Echo Api spec
- Swagger-ui should be able to make the requests.
- Check also the Documentation page in the Developer Portal

**Special notes for your reviewer**:

Blocked by https://github.com/3scale/platform/issues/434
